src/python/turicreate/test/test_drawing_classifier.py

```diff
@@ -97,7 +97,7 @@ def setUpClass(self, warm_start='auto'):
                 feature=self.feature,
                 max_iterations=1,
                 warm_start=warm_start)
-        self.trains = [self.check_cross_sf, self.stroke_sf]
+        self.trains = [self.check_cross_sf] # , self.stroke_sf]
```

Comment:
probably want to delete the commented out code here and on the next line.

Reply:
Yes, once `stroke` based tests are all enabled, we will.